docs/base/polizas.md

```diff
@@ -321,6 +321,62 @@ flux d'aquesta.
 
 ![](_static/polizas/historic_estats.png)
 
+## Contractes de tipus eventual
+
+Un contracte eventual és un tipus de contracte amb una duració determinada,
+aquests es solen realitzar per events de curta duració, com poden ser
+fires, certàmens, etc.
+
+Dins d'aquests contractes en podem distingir dos tipus:
+
+* **Contracte eventual sense comptador**: aquests són contractes d'una duració
+màxima de dos mesos en els quals per llei no fa falta un comptador, el consum es
+calcularà a partir d'una fórmula (Potència contractada * dies de contracte * hores diaries).
+
+* **Contracte eventual amb comptador**: a diferencia del contracte anterior aquest
+sí que disposa de comptador per poder calcular el consum.
+
+## Càlcul del consum total per contractes eventuals sense comptador
```

Comment:
Better to use header 3 (###) and include this section inside of "Contractes de tipus eventual"